src/utils/date.ts

```diff
@@ -76,6 +76,25 @@ export function parseDate(
   return isValid(parsedDate) && strictParsingValueMatch ? parsedDate : null;
 }
 
+export function getDefaultDateFormat() {
+  const formatObj = new Intl.DateTimeFormat().formatToParts(new Date());
```

Comment:
Did you test it in IE11?

Reply:
https://caniuse.com/#search=DateTimeFormat says it has to work but I will double-check it.

Reply:
right, but not `formatToParts` https://caniuse.com/#search=formatToParts